docs/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: typestubs pre watch dist settings-schema
+
+clean:
+	rm -rf docs/_build/
+	rm -rf docs/api/napari*.rst
+	rm -rf docs/gallery/*
+
+docs:
```

Comment:
Do we want to keep `pip install -qr docs/requirements.txt` here?

Reply:
I would say no - I (just before this PR) opened an issue at https://github.com/napari/napari/issues/4451 with my reasoning.